bin/i18n/resources/animations.rb

```diff
@@ -0,0 +1,23 @@
+require 'fileutils'
+require 'json'
+
+require_relative '../i18n_script_utils'
+require_relative '../../animation_assets/manifest_builder'
+
+module I18n
+  module Resources
+    module Animations
```

Comment:
would this be inside DAshboard, Apps or PEgasus?

Reply:
I'm not sure, since it has its own folder in `i18n/locales/source`, I thought it should be on the same level as the rest.
Where do you propose to move it then?
![Screenshot 2023-07-12 at 14 01 38](https://github.com/code-dot-org/code-dot-org/assets/11708250/521f2d6e-4a71-45af-a503-df7e57534f0c)